src/components/Results.css

```diff
@@ -9,3 +9,8 @@
 .no-result {
   color: rgb(43, 43, 43);
 }
+
+.no-result-text {
+  color: var(--text-color-first);
+  transition: 0.4s;
```

Comment:
I think we dont need to create another class. `.no-result` css rule can be reused !

Reply:
Yeah, I missed this trick. Fixing this in a moment @faran4engg

Reply:
Done @faran4engg 
You can review the code